bertopic/_bertopic.py

```diff
@@ -3688,6 +3691,8 @@ def _combine_zeroshot_topics(self,
         reverse_topic_labels = dict((v, k) for k, v in merged_model.topic_labels_.items())
         df.Label = df.Label.map(reverse_topic_labels)
         merged_model.topics_ = df.Label.values
+        if self._outliers:
+            reverse_topic_labels["Outliers"] = -1
```

Comment:
Aside: I think that the way this is set up means that the values in `merged_model.topics_` have data type [`Int64`](https://pandas.pydata.org/docs/user_guide/integer_na.html), because initially `df.Label` has integers and `nan` values in them, before the `nan`s are replaced with `-1`.

Not a big issue but it means that the data type for topics is different if you merge models.

Reply:
Awesome! Thanks for looking into this, it is highly appreciated. I made both changes you suggested which, hopefully, should have resolved this issue. If these changes indeed fix the underlying issue, most likely I will create a new minor release (0.16.2) considering zero-shot topic modeling in BERTopic is widely used.